debian/entry.sh

```diff
@@ -90,6 +90,8 @@ function init_non_systemd()
 		"$CMD" "$@" &
 		pid=$!
 		wait "$pid"
+		exit_code=$?
+		fg > /dev/null || exit "$exit_code"
```

Comment:
@petrosagg I've update the commit message and https://github.com/resin-io-library/base-images/issues/250 to explain more about this issue.